src/vision/oak/camera_oak.py

```diff
@@ -85,13 +88,25 @@ def get_data(self):
                 self.line_points.append(pose[:3, 3])
                 #pcd.transform(pose)
 
+            print("1",pcd)
+
+
+            if not pcd.is_empty():
+                pcd = pcd.voxel_down_sample(voxel_size=100)
+                #pcd.transform(pose)
+                pcd = assignment_to_sectors(pcd)
+            print("2",pcd)
             if self.visualize:
                 cvRGBFrame = cv2.cvtColor(cv_color_frame, cv2.COLOR_BGR2RGB)
                 colors = (cvRGBFrame.reshape(-1, 3) / 255.0).astype(np.float64)
                 pcd.colors = o3d.utility.Vector3dVector(colors)
                 self.i += 1
                 if self.i > 10:
                     self.vis.add_geometry(pcd)
+                if self.i > 50:
+                    self.vis.run()
+                    while True:
+                        pass
```

Comment:
magiczne wartości, ciężko wyłapać co znaczą bez kontekstu

Reply:
Do jest Daniela do wizualizacji.

Reply:
tego nie było na main służyło do testów xD

Reply:
proponuje niech to na razie zostanie ja dodam zmienną startup_iterations  i przycisk do obsłogi tego ze swojego brancha

Reply:
@Kleczyk @FWalkowicz pamiętajcie o dokumentowaniu wartości jakie wstawiacie :v na ten moment robię resolved